manifests/charts/gateways/istio-egress/templates/zzz_profile.yaml

```diff
@@ -41,3 +41,19 @@ Finally, we can set all of that under .Values so the chart behaves without aware
 {{- $a := mustMergeOverwrite $defaults $globals  }}
 {{- end }}
 {{- $b := set $ "Values" (mustMergeOverwrite $defaults $.Values) }}
+
+{{/*
+Labels that should be applied to ALL resources.
+*/}}
+{{- define "istio.labels" -}}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/part-of: istio
+app.kubernetes.io/version: {{ .Chart.AppVersion }}
+helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+{{- range $key, $val := .Values.labels }}
+{{- if and (ne $key "app") (ne $key "istio") }}
```

Comment:
How important are these guards? They only appear half the time

Reply:
All they're doing is quoting values for user-custom labels.

If we leave user-custom labels out of the template for now as I suggest in other comments, we can elide this problem for now.


(I think they're useless TBH - if the templates render, quoting shouldn't matter, and if they don't render with your custom labels because they're not quoted properly, fix your custom labels and re-render)